packages/tokens/build.js

```diff
@@ -43,11 +43,16 @@ const stripModeReducer = (result, token) => {
   return result
 }
 
-/** Filter function to return tokens of category 'font', type from filter, and npm true */
+/** Filter function to return tokens of category 'font', type from filter, and npm true. Exclude web font tokens */
 const filterFont = (token, fontType) => {
   const { category, type, npm } = token.attributes
 
-  return category === 'font' && type === fontType && npm === true
+  return (
+    category !== 'vads-font' &&
+    category === 'font' &&
```

Comment:
Isn't this redundant? If the category is equal to `font`, it's definitely not `vads-font`.

If we did want to include it to somewhat document in the code, maybe instead do it as a separate return of: `if (category === 'vads-font') return false` and leave the original return relating to our area of interest as it was.